android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -1169,6 +1170,59 @@ class JsonConverter {
             playerViewConfig = toPlayerViewConfig(json),
             pictureInPictureConfig = toPictureInPictureConfig(json.getMap("pictureInPictureConfig")),
         )
+
+        @JvmStatic
+        fun fromBufferLevel(bufferLevel: BufferLevel?): WritableMap? {
+            if (bufferLevel == null) {
+                return null
+            }
+
+            return Arguments.createMap().apply {
+                putDouble("level", bufferLevel.level)
+                putDouble("targetLevel", bufferLevel.targetLevel)
+                putInt(
+                    "media",
+                    when (bufferLevel.media) {
+                        MediaType.Audio -> 0
+                        MediaType.Video -> 1
+                    },
+                )
+                putInt(
+                    "type",
+                    when (bufferLevel.type) {
```

Comment:
Same as above

Reply:
Thanks for the comment, yeah using Strings makes a lot more sense.
Address in 6e7ca13d6f669ddf79ec4c777b71ddbff453bf7d